apps/scratch/src/types/scratch-student-activities/common.ts

```diff
@@ -0,0 +1,41 @@
+import { Block } from "scratch-blocks";
+import { CrtContextValue } from "../../contexts/CrtContext";
+import { WorkspaceChangeEvent } from "../scratch-workspace";
+
+export interface BaseStudentAppActivity extends Record<string, unknown> {
+  blockId: string;
+  blockType: string;
+}
+
+export interface BaseActivityRequest {
```

Comment:
No need to prefix this with `Base`

```suggestion
export interface ActivityRequest {
```